notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<ul><li>Something off with this sentence: "you can use just follow the model's url"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Re-phrased</p>